cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -231,8 +246,8 @@ impl Inst {
         let lower_reg = to_regs.regs()[0];
         let upper_reg = to_regs.regs()[1];
 
-        let mut load_ins = Inst::load_constant(lower_reg, lower);
-        let load_upper = Inst::load_constant(upper_reg, upper);
+        let mut load_ins = Inst::load_constant(lower_reg, lower, &mut alloc_tmp);
+        let load_upper = Inst::load_constant(upper_reg, upper, &mut alloc_tmp);
```

Comment:
The two consecutive calls to `load_constant` are what's forcing the `alloc_tmp` argument to `load_constant` to be a mutable reference.